pkg/controller/common/container/container.go

```diff
@@ -88,7 +89,14 @@ func ImageRepository(img Image, ver version.Version) string {
 	}
 	// Starting with 9.x ubi is the default for APM server, there's no -ubi image.
 	if useUBISuffix && ver.Major >= 9 {
-		if img == APMServerImage {
+		switch img {
+		case APMServerImage,
+			AgentImageSince9,
+			FilebeatImage,
+			MetricbeatImage,
+			HeartbeatImage,
+			AuditbeatImage,
+			PacketbeatImage:
```

Comment:
Should this not affect all stack images? Are there ones that keep the `-ubi` suffix?

Reply:
I am 100% sure only for elastic-agent and beats 🙂 That said I can't find anything about Journalbeat 

```
$ docker manifest inspect docker.elastic.co/beats/journalbeat:9.0.0-SNAPSHOT
no such manifest: docker.elastic.co/beats/journalbeat:9.0.0-SNAPSHOT
```

Reply:
Journalbeat was removed in 7.16 to my knowledge

Reply:
ok that makes sense, then everything I am 100% sure that needs to change is in this PR 🙂